tests/artemis-test-support/src/main/java/org/apache/activemq/transport/amqp/client/AmqpMessage.java

```diff
@@ -640,6 +640,11 @@ public String getText() throws NoSuchElementException {
       throw new NoSuchElementException("Message does not contain a String body");
    }
 
+   public byte[] getBytes() {
+      Data body = (Data)getWrappedMessage().getBody();
+      return body.getValue().getArray();
```

Comment:
The byte array contained within is not guaranteed to be zero indexed or span the full array, you need to check and account for that if you are going to expose it.

Reply:
I just wanted to assert a few bytes[] that were set with setBytes([]).

I have no idea on what I'm supposed to do here. I will just revert the test addition.